RELEASE_PROCESS.md

```diff
@@ -0,0 +1,36 @@
+# Building a Compose release
+
+## Building binaries
+
+`script/build-linux` builds the Linux binary inside a Docker container:
+
+    $ script/build-linux
+
+`script/build-osx` builds the Mac OS X binary inside a virtualenv:
+
+    $ script/build-osx
+
+For official releases, you should build inside a Mountain Lion VM for proper
+compatibility. Run the this script first to prepare the environment before
```

Comment:
Saw some other minor things in this file, but that would be wasting time if it will be replace by the text @moxiegirl wrote up :smile: